examples/with-urql/pages/pokemon/[name].js

```diff
@@ -0,0 +1,33 @@
+import { getPokemon } from '../../graphql/getPokemon'
+import { getPokemons } from '../../graphql/getPokemons'
+
+export default ({ pokemon }) => {
```

Comment:
This must be a named component:
```suggestion
export default function Pokemon({ pokemon }) {
```